### PR TITLE
[MSG] Use new RN inquiry modal from artwork VC

### PR DESCRIPTION
**Pull request by mennenia**
This can't be merged yet ofc, because it relies on an Emission update, but when that's good to go so should this be! :)

### CODE REVIEW

**Comment by mennenia** [comment]
@alloy @orta the beta says I added new routing for inquiries when this hasn't been merged (or same work added to PR #2379 e.g.) 😅

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Could not find the commit 54a8bd7e6436b0225e18956bb7bd4243aed940f2 in <a href="https://github.com/artsy/eigen-artefacts/">artsy/eigen-artefacts</a>.</p>

<p>You need to <code>git push</code> in the Pods dir.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_travis">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m [hunk 1]
```diff
@@ -106,8 +107,8 @@ - (void)tappedContactGallery
         return;
     }
 
-    ARInquireForArtworkViewController *inquireVC = [[ARInquireForArtworkViewController alloc] initWithPartnerInquiryForArtwork:self.artwork fair:self.fair];
-    [inquireVC presentFormWithInquiryURLRepresentation:[self inquiryURLRepresentation]];
+    ARInquiryComponentViewController *inquireVC = [[ARInquiryComponentViewController alloc] initWithArtworkID:self.artwork.artworkID];
+    [[ARTopMenuViewController sharedController] presentViewController:inquireVC animated:YES completion:nil];
```
Comment (alloy) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: Are we losing anything important with the removal of the ‘fair’ context?

Comment (mennenia) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: It doesn't seem like it. In `ARInquireForArtworkViewController` the `fair` property is being set, but then no longer used anywhere else in the class. There are no fairs on at the moment (can't we always have a fake fair on staging? :o), so I can't necessarily test in the flesh.

Comment (alloy) [inline Artsy/View_Controllers/Artwork/ARArtworkViewController+ButtonActions.m]: Ok :+1:

**Pull request merged**